examples/ota-requestor-app/ota-requestor-common/BDXDownloader.cpp

```diff
@@ -107,11 +107,17 @@ void BdxDownloader::HandleTransferSessionOutput(TransferSession::OutputEvent & e
     case TransferSession::OutputEventType::kTransferTimeout:
         ChipLogError(BDX, "Transfer timed out");
         mTransfer.Reset();
+        mExchangeCtx->Close();
```

Comment:
I think you need to add this to `kStatusReceived` and `kInternalError` as well, since both events signal the end of a transfer and will not result in more messages being sent.

Reply:
Fixed